src/components/RangeInput.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div :class="bem()">
+    <slot name="header" />
+
+    <form @submit.prevent="onSubmit(refinement)">
+      <input
+        type="number"
+        :class="bem('input', 'from')"
+        :min="range.min"
+        :max="range.max"
+        :step="step"
+        :placeholder="rangeForRendering.min"
+        :value="refinementForRendering.min"
+        @input="refinement.min = $event.target.value"
+      />
+
+      <slot name="separator">
+        to
+      </slot>
+
+      <input
+        type="number"
+        :class="bem('input', 'to')"
+        :min="range.min"
+        :max="range.max"
+        :step="step"
+        :placeholder="rangeForRendering.max"
+        :value="refinementForRendering.max"
+        @input="refinement.max = $event.target.value"
+      />
+      <slot name="submit">
+        <button>Ok</button>
```

Comment:
We might want to add a class here, wdyt?

Reply:
I don't added it because it's a default placeholder. If the user want to modify it, he can override the slot. But why not, it can be a quick win for the customisation of the component.